flake-info/src/elastic.rs

```diff
@@ -155,14 +159,26 @@ lazy_static! {
                         "token_chars": [
                             "letter",
                             "digit",
-                            // Either we use them or we would need to strip them before that.
                             "punctuation",
-                            "symbol",
+                            "custom",
                         ],
+                        // Exclude XML characters < and > from "symbol"
+                        "custom_token_chars": "+=~",
+                    },
+                    "attr_path": {
+                        "type": "path_hierarchy",
+                        "delimiter": ".",
+                    },
+                    "attr_path_reverse": {
+                        "type": "path_hierarchy",
+                        "delimiter": ".",
+                        "reverse": true,
                     },
```

Comment:
Didn't we remove those reverse fields recently or is this just allowing us to search attrs in paths from right to left?

Reply:
The reverse fields we had were string reversals computed in Rust, e.g. turning `foo.bar` into `rab.oof` so that a search for `bar` could be reversed to `rab` and then match the prefix generated by `edge_ngram`.

This is a different approach, `reverse` here just means we start splitting the string from the right so that we end up with `bar` and `foo.bar`.